documentation/disaster-recovery.md

```diff
@@ -86,3 +93,29 @@ cf service teaching-vacancies-postgres-production
  Once the database has been successfully restored, the corrupted database instance should be deleted.
 
  `cf delete-service teaching-vacancies-<env>-old -f`
+
+## Loss of database instance
+
+In case the database instance is lost, the objectives are:
+
+- Recreate the lost postgress database instance
+- Restore data from daily backup
+
+### Recreate the lost postgress database instance
+
+In order to recreate the lost postgress database instance, use the following make recipes `terraform-app-plan` and `terraform-app-apply`:
+
+```
+make <env> terraform-app-plan passcode=MyPasscode tag=master-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714 ## to see the proposed changes.
```

Comment:
We could leave the tag out so it uses the default
BTW this must be changed here: https://github.com/DFE-Digital/teaching-vacancies/blob/main/Makefile#L110

Reply:
since it is production, don't we want to use exactly same tag?

Reply:
OK then please show how to retrieve the tag